create/how_to/aws.md

```diff
@@ -74,7 +74,7 @@ For your MeiliSearch instance to communicate with the outside world, it is very
 - Click on **Add rule** and select **HTTP** from the drop-down menu. This will open the HTTP port (80).
 - Click on **Add rule** and select **HTTPS** from the drop-down menu. This will open the HTTPS port (443).
 
-![Security Groups: Networking and connectivity](/aws/07.security.png)
+![Adding HTTP rules to the instance](/aws/07.security.png)
```

Comment:
May want to mention the SSH security group since it's not covered in the text. Also the fact that Source is set to "Anywhere" by default should be mentioned here, since it's unclear without the image.